README.md

```diff
@@ -24,7 +24,7 @@ bower install tiny-emitter --save
 
 ## Usage
 
-Node and Browserify
+### Node and Browserify
```

Comment:
Probably change this to `Node and Browser` as this can be used with more than Browserify. It's a relic from before webpack was a thing.